core/complexStructures/processedMiniBlocks.go

```diff
@@ -0,0 +1,132 @@
+package complexStructures
```

Comment:
please rename the directory and the package name.

Reply:
also I think you can move this file to the `process/block` directory because it belongs there and this way you will get rid of this bad naming.

Reply:
done